WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -1940,4 +1942,11 @@ protected void copyDatabase() {
     public boolean hasAnyJetpackBlogs() {
         return SqlUtils.boolForQuery(db, "SELECT 1 FROM " + BLOGS_TABLE + " WHERE api_blogid != 0 LIMIT 1", null);
     }
+
+    /*
+     * This is a stub method. Once implemented it'll retrive a person object for the provided local id
+     */
+    public Person getPersonForLocalTablePostId(long localTablePersonId) {
+        return new Person(4, "oguzkocer", "Oguz", "Kocer", "Oguz", "http://lorempixum.com/76/76", Role.EDITOR);
+    }
```

Comment:
I know it's only a stub method but we should not add anything to `WordPressDB`. Please create a "PersonTable" class and put all db related method there.